### ISSUE TITLE
0.5.1 RC2 Docker no http server

### ISSUE THREAD

**Issue by mastershake8241** [blakeblackshear/frigate#116]
No web pages come up and doing a ps -A shows all the http processes as defunct. 

0.5.1 RC1 Seems to "work" except no picture comes up for any camera when going to the http://ipaddress:5000/back or /back/person/best.jpg

**Comment by blakeblackshear** [comment]
What do the logs show? Have previous versions worked for you?

**Comment by mastershake8241** [comment]
Not much. It's not showing that flask is starting because it's not showing the message about not running it in production. It's not showing the usual output when it's connecting to/setting up the cameras in the config.

> On connect called
> /arrow/cpp/src/plasma/store.cc:1226: Allowing the Plasma store to use up to 0.4GB of memory.
> /arrow/cpp/src/plasma/store.cc:1253: Starting object store with directory /dev/shm and huge >page support disabled
>
>ffprobe -v panic -show_error -show_streams -of json >"rtsp://admin:@192.168.1.169:554/h264Preview_01_main"
>Starting detection process: 22
>
>{'streams': [{'index': 0, 'codec_name': 'h264', 'codec_long_name': 'H.264 / AVC / MPEG-4 AVC / >MPEG-4 part 10', 'profile': 'High', 'codec_type': 'video', 'codec_time_base': '0/2', 'codec_tag_string': >'[0][0][0][0]', 'codec_tag': '0x0000', 'width': 1920, 'height': 1072, 'coded_width': 1920, 'coded_height': >1072, 'has_b_frames': 0, 'sample_aspect_ratio': '0:1', 'display_aspect_ratio': '0:1', 'pix_fmt': 'yuv420p', >'level': 40, 'chroma_location': 'left', 'field_order': 'progressive', 'refs': 1, 'is_avc': 'false', >'nal_length_size': '0', 'r_frame_rate': '30/1', 'avg_frame_rate': '0/0', 'time_base': '1/90000', 'start_pts': >5940, 'start_time': '0.066000', 'bits_per_raw_sample': '8', 'disposition': {'default': 0, 'dub': 0, 'original': >0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, >'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}}, {'index': 1, 'codec_name': 'aac', >'codec_long_name': 'AAC (Advanced Audio Coding)', 'profile': 'LC', 'codec_type': 'audio', >'codec_time_base': '1/16000', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'sample_fmt': >'fltp', 'sample_rate': '16000', 'channels': 1, 'channel_layout': 'mono', 'bits_per_sample': 0, >'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/16000', 'start_pts': 0, 'start_time': >'0.000000', 'disposition': {'default': 0, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, >'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, >'timed_thumbnails': 0}}]}
>
>eating ffmpeg process...
>
>ffmpeg -hide_banner -loglevel panic -avoid_negative_ts make_zero -fflags nobuffer -flags >low_delay -strict experimental -fflags +genpts+discardcorrupt -vsync drop -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://admin:@192.168.1.169:554/h264Preview_01_main -f rawvideo -pix_fmt rgb24 pipe:
ffprobe -v panic -show_error -show_streams -of json "rtsp://admin:@192.168.1.38:554/h264Preview_01_main"
{'streams': [{'index': 0, 'codec_name': 'h264', 'codec_long_name': 'H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10', 'profile': 'High', 'codec_type': 'video', 'codec_time_base': '0/2', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'width': 1920, 'height': 1072, 'coded_width': 1920, 'coded_height': 1072, 'has_b_frames': 0, 'sample_aspect_ratio': '0:1', 'display_aspect_ratio': '0:1', 'pix_fmt': 'yuv420p', 'level': 40, 'chroma_location': 'left', 'field_order': 'progressive', 'refs': 1, 'is_avc': 'false', 'nal_length_size': '0', 'r_frame_rate': '30/1', 'avg_frame_rate': '0/0', 'time_base': '1/90000', 'start_pts': 2970, 'start_time': '0.033000', 'bits_per_raw_sample': '8', 'disposition': {'default': 0, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}}, {'index': 1, 'codec_name': 'aac', 'codec_long_name': 'AAC (Advanced Audio Coding)', 'profile': 'LC', 'codec_type': 'audio', 'codec_time_base': '1/16000', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'sample_fmt': 'fltp', 'sample_rate': '16000', 'channels': 1, 'channel_layout': 'mono', 'bits_per_sample': 0, 'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/16000', 'start_pts': 0, 'start_time': '0.000000', 'disposition': {'default': 0, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}}]}
>
>eating ffmpeg process...
>
>ffmpeg -hide_banner -loglevel panic -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental -fflags +genpts+discardcorrupt -vsync drop -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://admin:@192.168.1.38:554/h264Preview_01_main -f rawvideo -pix_fmt rgb24 pipe:
ffprobe -v panic -show_error -show_streams -of json "rtsp://admin:@192.168.1.175:554/h264Preview_01_main"
>
>/arrow/cpp/src/plasma/eviction_policy.cc:134: There is not enough space to create this object, so >evicting 13 objects to free up 80281344 bytes. The number of bytes in use (before this eviction) is >395231232.
>
>/arrow/cpp/src/plasma/eviction_policy.cc:134: There is not enough space to create this object, so >evicting 13 objects to free up 80281344 bytes. The number of bytes in use (before this eviction) is >395231232.

And those last two sections just are repeating

**Comment by blakeblackshear** [comment]
Is the resolution of your camera 1920x1072? Seems like a strange resolution. Can you post your config?

**Comment by mastershake8241** [comment]
I set the size of the resolution manually in the config and now it's starting up flask. I'm still getting the "There is not enough space to create this object...." errors. 

Now the image is distorted. 
![Annotation 2020-03-15 101645](https://user-images.githubusercontent.com/31140079/76706667-7282cb00-66a6-11ea-91ca-8ec24288e9c3.png)

Here's the config: (it wouldn't let me upload with .yml extension just fyi)
[config.txt](https://github.com/blakeblackshear/frigate/files/4334642/config.txt)

I did verify the streams work when opening with VLC.
